### PR TITLE
1 cloning

### PR DESCRIPTION
**Pull request by Rangtuk**

### CODE REVIEW

File: GitIQT/Program.cs [hunk 1]
```diff
@@ -11,6 +11,7 @@ static void Main(string[] args)
                 "======================================================================================================================",
                 "It's your first day at your new job and there's a lot of stuff to do. Your boss gives you a list of tasks to complete:",
                 "1. Clone a repository",
+                "    ( Clone from main branch or a specific branch ) ",
```
Comment (yurisim) [inline GitIQT/Program.cs]: what was the intent for this? The formatting seems off

File: GitIQT/Scenarios/ChangeDirectory.cs [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        private string Answer = string.Empty;
+
+        // Ask user to commit changes to their branch
+        public void AskPrompt()
+        {
+            // Ask user to commit changes to their branch
+            Answer = $"cd ProjectBacon";
+
+            var prompt = "However, while the repository has been cloned, you are not currently in it. What do you need to do next to access the local repository?";
+
```
Comment (yurisim) [inline GitIQT/Scenarios/ChangeDirectory.cs:16]: Tell the user what directory they are currently in and where they should be after a clone.

image

Comment (yurisim) [reply GitIQT/Scenarios/ChangeDirectory.cs:16]: Do they know that they should be in Project Bacon?

File: GitIQT/Scenarios/CheckBranch.cs [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+        private string Answer = string.Empty;
+
+        // Ask user to check their current branch
+        public void AskPrompt()
+        {
+            // Ask user to check their current branch
+            Answer = $"git branch";
+
+            var prompt = $"What git command do you need to type in to check the branch you are currently on?";
```
Comment (yurisim) [inline GitIQT/Scenarios/CheckBranch.cs:18]: allow `git status` to also work

File: GitIQT/Scenarios/CheckoutDev.cs [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+        private string Answer = string.Empty;
+
+        // Ask user to ckeckout dev branch
+        public void AskPrompt()
+        {
+            // Ask user to ckeckout dev branch
+            Answer = $"git checkout dev";
+
+            var prompt = $"What git command do you need to type in to checkout the repository named 'dev'?";
```
Comment (yurisim) [inline GitIQT/Scenarios/CheckoutDev.cs]: Typo,
image

File: GitIQT/Scenarios/CreateBranch.cs [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+        private string Answer = string.Empty;
+
+        // Ask user to create their own branch
+        public void AskPrompt()
+        {
+            // Ask user to create their own branch
+            Answer = $"git branch feature/feature-name";
+
+            var prompt = $"What git command do you need to type in to create a new branch from the dev branch? (Please name it feature/feature-name)";
```
Comment (yurisim) [inline GitIQT/Scenarios/CreateBranch.cs]: Allow 
image
 to also work

File: GitIQT/Scenarios/CommitChange.cs [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+
+        private string Answer = string.Empty;
+
+        // Ask user to commit changes to their branch
+        public void AskPrompt()
+        {
+            // Ask user to commit changes to their branch
+            Answer = $"git commit";
+
+            var prompt = $"What git command do you need to type in to commit changes to the branch? (git add has already been performed)";
```
Comment (yurisim) [inline GitIQT/Scenarios/CommitChange.cs:15]: image

You also need to do the commit message

**Review by yurisim — CHANGES_REQUESTED** [review]
Overall this was great! 

I think this will be a very useful tool for training. There's some alternative paths that you guys should add that are in this review, but after that's done this looks great :)

Keep it up!

File: GitIQT/Program.cs (hunk 1 above)
Comment (Rangtuk) [reply GitIQT/Program.cs]: Intent was to provide multiple options expect in the initial prompt, supposed to give the appearance of being indented thus still related to the 1st "Clone a repository" task. as the expectations on Issue #1 asked for cloning a generic branch (default master/main) or cloning from a specific branch

**Comment by yurisim** [comment]
image

Change text in this?

**Pull request merged**